source/dynk.f90

```diff
@@ -2519,6 +2601,16 @@ real(kind=fPrec) function dynk_getvalue(element_name, att_name)
       ! Not yet supported : AC dipole (16)
 
       ! Not yet supported : beam-beam separation (20)
+      case(20)
+       if (att_name == "h-sep") then ! [mm]
```

Comment:
Wrong indentation in this section. Should be two space indentation in all new code in SixTrack.